src/lib/journey/callbacks/text-output/text-output.mock.ts

```diff
@@ -11,3 +13,56 @@ export const docsExample = {
     },
   ],
 };
+
+export default {
+  authId: 'foo',
+  "callbacks": [
+    {
+      "type": "TextOutputCallback",
+      "output": [
+        {
+          "name": "message",
+          "value": "Had coffee?"
```

Comment:
Can we have multiple variants of this callback? Some with plaintext, some with HTML, some that are malicious code?